.github/actions/create-release-candidate-v1/action.yml

```
@@ -0,0 +1,228 @@
+name: create-release-candidate
+description: 'A GitHub Action to create a release candidate'
+
+inputs:
+  token:
+    description: 'A GitHub token used for octokit and GH CLI with the required permissions'
+    required: true
+  base:
+    description: 'The branch the pull request will be merged into'
+    required: true
+  head:
+    description: 'The branch that contains the changes the pull request is trying to merge'
+    required: true
+  release-script-path:
+    description: 'The location of the `prepare_release.sh` shell script that creates the changelogs and version bumps'
+    required: true
+  slack-webhook:
+    description: 'The Slack webhook to send the release candidate notification to'
+    required: true
+  version-locked:
+    description: 'Whether or not the version should be locked to axe-core'
+    default: 'false'
+    required: false
+  docs-repo:
+    description: 'The name of the repo where the release notes live'
+    default: 'null'
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Checkout repository
+      uses: actions/checkout@v4
+      with:
+        # Fetch all history
+        fetch-depth: 0
+
+    - name: Is release in progress
+      uses: dequelabs/axe-api-team-public/.github/actions/is-release-in-progress-v1@main
+      id: is_release_in_progress
+      with:
+        github-token: ${{ inputs.token }}
+
+    - name: Generate commit list
+      if: ${{ !steps.is_release_in_progress.outputs.in_progress }}
+      uses: dequelabs/axe-api-team-public/.github/actions/generate-commit-list-v1@main
+      id: generate_commit_list
+      with:
+        base: ${{ inputs.base }}
+        head: ${{ inputs.head }}
+
+    - name: Has releasable commits
+      if: ${{ !steps.is_release_in_progress.outputs.in_progress }}
+      uses: dequelabs/axe-api-team-public/.github/actions/has-auto-releasable-commits-v1@main
+      id: has_releasable_commits
+      with:
+        commits: '${{ steps.generate_commit_list.outputs.commit-list }}'
+        version-locked: ${{ inputs.version-locked }}
+
+    - name: Cancel release if a release is already in progress or there are no releasable commits
+      shell: bash
+      if: ${{ steps.is_release_in_progress.outputs.in_progress == 'true' || steps.has_releasable_commits.outputs.should_release == 'false' }}
+      run: |
+        echo "Release is in progress: ${{ steps.is_release_in_progress.outputs.in_progress }}"
+        echo "Should release: ${{ steps.has_releasable_commits.outputs.should_release }}"
+        gh run cancel ${{ github.run_id }}
+        gh run watch ${{ github.run_id }}
+
+    - name: Run prepare_release shell script
+      shell: bash
+      id: prepare_release
+      run: |
+        if [ "${{ inputs.version-locked }}" = "true" ]
+        then
+          # We will always release as patch if the version is locked to axe-core
+          bash ${{ inputs.release-script-path }} patch
+        else 
+          bash ${{ inputs.release-script-path }}
```

Should be able to run the script without calling `bash`. This will also let the script decide if it's running bash or node

```suggestion
          ${{ inputs.release-script-path }} patch
        else 
          ${{ inputs.release-script-path }}
```